cleaning_data.py

```diff
@@ -0,0 +1,199 @@
+"""change documents to UTF-8"""
+
+
+"""tohle má ostatní texty uložit do UTF-8 formátu
+Taky by to mělo vymazat všechno, kromě textu knihy ze zdrojů gutenberg"""
+"""CHANGE documents to UTF-8 and save as a new files and convert to one dictionary"""
+import os
+import glob
+
+def read_text_files_from_subfolders(main_folder_path):
+# Find all .txt files in the specified main folder and its subfolders

```

Comment:
this should be docstring
```
def read_text_files_from_subfolders(main_folder_path):
    """Find all .txt files in the specified main folder and its subfolders."""
```
Docstring in `"""` can be used to autogenerate documentation and are preferred over comments `#`